tests/scripts/remote_monitoring_tests/rest_apis/test_list_recommendations.py

```diff
@@ -651,6 +708,7 @@ def test_list_recommendations_multiple_exps_with_missing_metrics(cluster_type):
         response = delete_experiment(create_exp_json_file)
         print("delete exp = ", response.status_code)
 
+
 @pytest.mark.extended
 @pytest.mark.parametrize("latest", ["true", "false"])
 def test_list_recommendations_with_only_latest(latest, cluster_type):
```

Comment:
@msvinaykumar - rest_apis/test_list_recommendations.py::test_list_recommendations_with_only_latest[false] this test failed with the below error, can you please check

```
      # Validate the json against the json schema
        errorMsg = validate_list_reco_json(list_reco_json, list_reco_json_schema)
>       assert errorMsg == ""
E       assert [<ValidationError: "'monitoring_start_time' is a required property">, <ValidationError: "'monitoring_end_time' is a re...Error: "'variation' is a required property">, <ValidationError: "'monitoring_start_time' is a required property">, ...] == ''

test_list_recommendations.py:805: AssertionError
```